contributing.md

```diff
@@ -83,7 +83,14 @@ When working on the extension or checking out branches, use this to have it cons
 npm run watch # Listen for file changes and automatically rebuild
 ```
 
-Then load or reload it into the browser to see the changes (this does not happen automatically).
+Then load or reload it into the browser to see the changes (this does not happen automatically). If you'd like
+it to occur automatically than install firefox and:
+
+```sh
+npm run dev
+```
+
+This will load default profile (browser settings), but it will not keep the changes if made in session launched with the script.
```

Comment:
```suggestion
This will load the default profile (browser settings), but it will not keep the changes if made in session launched with the script.
```

Reply:
This sentence needs to be better written.